CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1608,9 +1626,9 @@ do -- Make the Inventory expand/collapse arrow (unless TopBar)
 			disableGamepadInventoryControl()
 		end
 
-		if InventoryFrame.Visible and GamepadEnabled then
+		if InventoryFrame.Visible then
```

Comment:
This is part of it, there is also an issue with GamepadEnabled being set to true due to funny reporting of the GamepadConnected events/functions. At any rate, it is listening to ButtonX (only exists on the gamepads)